packages/magento-customer/components/OrderDetails/OrderDetails.tsx

```diff
@@ -105,255 +43,230 @@ export function OrderDetails(props: OrderDetailsProps) {
-
+                <Typography>{payment_methods[0].name}</Typography>
                 {invoices && invoices?.length > 0 && (
-                  <Invoice>
+                  <Box
+                    className={classes.invoice}
+                    sx={(theme) => ({
+                      display: 'flex',
+                      justifyContent: 'flex-start',
+                      alignItems: 'center',
+                      color: theme.palette.primary.main,
+                    })}
+                  >
                     <IconSvg src={iconInvoice} size='small' />
                     {invoices?.[0]?.number}
-                  </Invoice>
+                  </Box>
                 )}
               </>
             )}
-          </div>
-        </div>
-
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Shipping address' />
-          </OrderDetailTitle>
-          <div>
-            <div>
-              {shipping_address?.firstname} {shipping_address?.lastname}
-            </div>
-            <div>{shipping_address?.street}</div>
-            <div>
-              {shipping_address?.postcode} {shipping_address?.city}
-            </div>
-            <div>
-              {shippingAddressRegion?.name}, {shippingAddressCountry?.full_name_locale}
-            </div>
-          </div>
-        </div>
-
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Billing address' />
-          </OrderDetailTitle>
-          <div>
-            <div>
-              {billing_address?.firstname} {billing_address?.lastname}
-            </div>
-            <div>{billing_address?.street}</div>
-            <div>
-              {billing_address?.postcode} {billing_address?.city}
-            </div>
-            <div>
-              {billingAddressRegion?.name}, {billingAddressCountry?.full_name_locale}
-            </div>
-          </div>
-        </div>
-      </OrderDetailsInnerContainer>
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Shipping address' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
+            <AddressMultiLine {...shipping_address} />
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Billing address' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            >
+              <AddressMultiLine {...billing_address} />
+            </SectionContainer>
+          </Box>
+        </Box>
+      </SectionContainer>
 
-      <TotalsContainer>
-        <TotalsRow>
-          <div>
+      <Box
+        className={classes.totalsContainer}
```

Comment:
Do we want to move the OrderTotals to a separate component and move the OrderItems to in between, so that we have this order?

1. OrderOverview
2. OrderItems
3. OrderTotals

Reply:
Yes, I've changed it around. It's more in line with the checkout now.